engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesInDbHelper.java

```diff
@@ -62,7 +62,7 @@ public abstract class EmailTemplatesInDbHelper {
     protected final Map<String, UUID> eventTypes = new HashMap<>();
 
     @BeforeEach
-    void initData() {
+    protected void initData() {
```

Comment:
Yes because this method have to be overrided on [TestSingleDailyTemplate](https://github.com/RedHatInsights/notifications-backend/pull/2234/files#diff-448183425f381f9f463444c5cfa2ced2b5c42e8703529bf6113e8a38ca3c701a), since we deal with a list of application instead of only one.